libs/libgpg-error/Makefile

```diff
@@ -91,6 +91,11 @@ define Build/InstallDev
 	$(INSTALL_DATA) \
 		$(PKG_INSTALL_DIR)/usr/share/aclocal/gpgrt.m4 \
 		$(1)/usr/share/aclocal/
+
+	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib/pkgconfig
```

Comment:
Could you please replace `$(STAGING_DIR) `with `$(1)`. This will add the file to the package list and not copy it directly to the staging directory. The advantage is that during development, when a package clean-up call is made, all installed files are properly cleaned up from the staging directory for this packages
Take a look at the openwrt buildsystem https://github.com/openwrt/openwrt/blob/master/include/package.mk#L231

Reply:
I pushed this change.

Reply:
> I pushed this change.

Really? Looks like nothing happended. Forgot '--force'?

Reply:
I did indeed forget `--force`. I just pushed again.